### PR TITLE
[DEV-4670] Uplift the question edit and answer edit pages to the new design

### PR DESCRIPTION
**Pull request by empanda**
https://roverdotcom.atlassian.net/browse/DEV-4671 and https://roverdotcom.atlassian.net/browse/DEV-4670

### CODE REVIEW

**Comment by empanda** [comment]
@kevinvanderlugt please review and +1. Thanks.

**Comment by empanda** [comment]
@kevinvanderlugt let me know if you need help setting up our branch of askbot in your development environment.

**Comment by kevinvanderlugt** [comment]
@empanda Yeah I will probably need some help getting this setup locally

**Comment by empanda** [comment]
Ok, lets tackle that Monday morning.

File: askbot/templates/question_edit.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     {% if revision_form.revision.errors %}{{ revision_form.revision.errors.as_ul() }}{% endif %}
     {{ revision_form.revision }}
     <input type="hidden" id="select_revision" name="select_revision" value="false"/>
-    <div class="form-item">
+    <div class="form-group">
         <label for="id_title" >
             {{ settings.WORDS_QUESTION_IN_ONE_SENTENCE|escape }}
```
Comment (kevinvanderlugt) [inline askbot/templates/question_edit.html]: We should use the same title "What is your question?" instead of the confusing settings.

**Comment by kevinvanderlugt** [comment]
:+1:   Pending the small title change

**Pull request merged**